include/fmt/compile.h

```diff
@@ -568,14 +594,35 @@ format_to_n_result<OutputIt> format_to_n(OutputIt out, size_t n,
   return {it.base(), it.count()};
 }
 
-template <typename CompiledFormat, typename... Args>
+template <typename CompiledFormat, typename... Args,
+          FMT_ENABLE_IF(internal::is_compiled_format<CompiledFormat>::value)>
 std::size_t formatted_size(const CompiledFormat& cf, const Args&... args) {
   return format_to(
              internal::counting_iterator<typename CompiledFormat::char_type>(),
              cf, args...)
       .count();
 }
 
+template <typename CompiledFormat, typename... Args,
+          FMT_ENABLE_IF(internal::is_compiled_format<CompiledFormat>::value)>
+inline void print(std::FILE* f, const CompiledFormat& cf, Args&&... args) {
```

Comment:
Please don't add `print` here and below, because duplicating formatting functions won't scale. In the future we'll make the original formatting functions work with compiled strings.

Reply:
It's probably better to handle this in a separate PR.